doc/tools/west/index.rst

```diff
@@ -226,7 +254,10 @@ to do so by manually creating an installation:
 
    # cd into zephyrproject if not already there
    git clone https://github.com/zephyrproject-rtos/west.git
-   echo > west/.west_topdir
+   echo > .west/config
+   # edit .west/config so that it contains:
+   [manifest]
+   path = zephyr
```

Comment:
How about just this instead (tested on windows / linux):
```
echo [manifest] > .west/config
echo path = zephyr >> .west/config
```

Reply:
Nice! i will add that

Reply:
Thanks!